src/js/visual/configurator-csr.js

```diff
@@ -529,6 +529,9 @@ ripe.ConfiguratorCsr.prototype.syncFromPRC = async function(prcConfigurator) {
     // sets the CSR configurator visuals so it matches the PRC frame
     const frame = ripe.getFrameKey(prcConfigurator.view, prcConfigurator.position);
     await this.changeFrame(frame, { duration: 0 });
+
+    // resets camera zoom to starting value
+    this._setZoom(1);

```

Comment:
@NFSS10 didnt you tell me this zoom value can vary? in other words, that 1 may not be the starting value?

Reply:
No, I told you the min and max value for the zoom can vary. 1 == no zoom